daras_ai_v2/base.py

```diff
@@ -1673,18 +1673,15 @@ def publish_and_redirect(self) -> typing.NoReturn | None:
 
         updated_count = SavedRun.objects.filter(
             id=self.current_sr.id,
-            uid=self.current_sr.uid,
             # filter for RecipeRunState.standby
             run_status="",
             error_msg="",
             run_time=datetime.timedelta(),
         ).update(run_status=STARTING_STATE, uid=self.request.user.uid)
         if updated_count >= 1:
             # updated now
-            self.call_runner_task(self.current_sr)
-        elif self.get_run_state(self.current_sr.to_dict()) == RecipeRunState.standby:
-            # updated by a different thread, we just refresh_from_db
```

Comment:
this was not necessary because we create a published run and redirect to that. we don't need `run_id` and `uid` to be correct in the query params for the link to work